### DISCUSSION TITLE
Expo compatibility

### DISCUSSION THREAD

**Discussion by retrx0** [react-native-google-cast/react-native-google-cast#284] [Q&A] [answered]
Hi everyone,I'm new to react-native just wanted to ask if this will work with expo, thanks.

**Comment by petrbela** [comment, marked as answer]
~As this library requires native dependencies that are not available in the default Expo bundle, you'll need to [eject to bare workflow](https://docs.expo.io/bare/customizing/). That will enable you to continue using Expo for the parts of the app that don't need casting, and then test Cast-specific functionality in a standalone build.~

It's no longer needed to eject to bare workflow. Expo 43 now supports custom native modules, see https://github.com/expo/config-plugins/tree/master/packages/react-native-google-cast for how to configure it.
